trk/src/Screens/ClimbDetail/index.js

```diff
@@ -1,8 +1,32 @@
-import React from 'react';
-import { View, Text, StyleSheet, SafeAreaView } from 'react-native';
+import React, {useState, useEffect } from 'react';
+import { View, Text, StyleSheet, SafeAreaView, Image } from 'react-native';
+import storage from '@react-native-firebase/storage';
 
 function ClimbDetail(props) {
   const { climbData } = props.route.params;
+  const [climbImageUrl, setClimbImageUrl] = useState(null);
+  const [setterImageUrl, setSetterImageUrl] = useState(null);
+
+  useEffect(() => {
+    const climbReference = storage().ref('climb photos/the_crag.png');
+    climbReference.getDownloadURL()
+      .then((url) => {
+        setClimbImageUrl(url);
+      })
+      .catch((error) => {
+        console.error("Error getting climb image URL: ", error);
+      });
+    
+    const setterReference = storage().ref('profile photos/epset.png');
+    setterReference.getDownloadURL()
+      .then((url) => {
+        setSetterImageUrl(url);
+      })
+      .catch((error) => {
+        console.error("Error getting setter image URL: ", error);
+      });
+  }, []);
+ 
```

Comment:
this is temporary as this file will be moved into an api call file so that every time a climb is read, the path is read in firestore in the climb collection imageRef key-value pair, and the image is downloaded for the user